enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +274,21 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
```

Comment:
nit: I'd leave out "and serve them when requested." I think that should be apparent, and we have a specific example below.